src/main/resources/paas.yaml

```diff
@@ -1,42 +0,0 @@
-database:
```

Comment:
I wonder if it is worth keeping this file in git just for versioning reasons. If you disagree perhaps enable versioning on the file in S3 so there is a means to track changes?